osu.Game.Rulesets.Osu/Scoring/OsuHealthProcessor.cs

```diff
@@ -115,6 +115,7 @@ private double getHealthIncreaseFor(JudgementResult result)
                 case HitResult.Great:
                     return 0.03;
 
+                case HitResult.IgnoreHit:
```

Comment:
This does more than intended because

https://github.com/ppy/osu/blob/16c7c14602c4575c302e3495587c5b7f23369e4d/osu.Game.Rulesets.Osu/Objects/Slider.cs#L278-L282
https://github.com/ppy/osu/blob/16c7c14602c4575c302e3495587c5b7f23369e4d/osu.Game.Rulesets.Osu/Judgements/OsuIgnoreJudgement.cs#L10

So now presumably sliders will give a small bonus to health with classic mod active that was not there before.

Reply:
Hmm, this is a fair point, though I'm not sure what the best way to solve this would be.
Adding a new HitResult for this would IMO be undesired.
Would it perhaps make sense to change `OsuIgnoreJudgement` to always have `HitResult.IgnoreMiss` as the end result?

Reply:
> Would it perhaps make sense to change `OsuIgnoreJudgement` to always have `HitResult.IgnoreMiss` as the end result?

It would not. It would die at this call.

https://github.com/ppy/osu/blob/16c7c14602c4575c302e3495587c5b7f23369e4d/osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs#L698

> Adding a new HitResult for this would IMO be undesired.

We already have

https://github.com/ppy/osu/blob/16c7c14602c4575c302e3495587c5b7f23369e4d/osu.Game/Rulesets/Scoring/HitResult.cs#L142-L147

so "undesired" is no longer a factor, the precedent has been broken. It's probably a new hit result or nothing.

Reply:
> > Would it perhaps make sense to change `OsuIgnoreJudgement` to always have `HitResult.IgnoreMiss` as the end result?
> 
> It would not. It would die at this call.
> 
> https://github.com/ppy/osu/blob/16c7c14602c4575c302e3495587c5b7f23369e4d/osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs#L698
> 
> > Adding a new HitResult for this would IMO be undesired.
> 
> We already have
> 
> https://github.com/ppy/osu/blob/16c7c14602c4575c302e3495587c5b7f23369e4d/osu.Game/Rulesets/Scoring/HitResult.cs#L142-L147
> 
> so "undesired" is no longer a factor, the precedent has been broken. It's probably a new hit result or nothing.

Okay, I will go that direction then 👍🏻